docs-conceptual/azps-6.4.0/what-is-azure-powershell.md

```diff
@@ -0,0 +1,55 @@
+---
+title: What is Azure PowerShell
+description: This article is an introduction to Azure PowerShell and its features.
+ms.devlang: powershell
+ms.topic: conceptual
+ms.date: 09/13/2021
+ms.custom: devx-track-azurepowershell
+ms.service: azure-powershell
+---
+
+# What is Azure PowerShell?
+
+Azure PowerShell is a set of cmdlets for managing Azure resources directly from PowerShell. Azure
+PowerShell is designed to make it easy to learn and get started with, but provides powerful features
+for automation.
```

Comment:
I think that we need to help someone new to Azure PowerShell to understand some fundamentals:
- Azure PowerShell make REST calls to Azure API.
- Management plane vs data plane
- How authentication works across modules & cmdlets
- Benefits of a cmdlet vs a direct REST call
- Maybe a touch on output objects